google-cloud-spanner/acceptance/spanner_helper.rb

```diff
@@ -31,16 +31,12 @@ def SecureRandom.int64
   random_bytes(8).unpack("q")[0]
 end
 
-def emulator_host
-  ENV["SPANNER_EMULATOR_HOST"]
-end
-
 def emulator_enabled?
-  !emulator_host.nil?
+  !ENV["SPANNER_EMULATOR_HOST"].nil?
```

Comment:
I think can just use 

```ruby
def emulator_enabled?
  ENV["SPANNER_EMULATOR_HOST"]
end
```

https://stackoverflow.com/questions/4012775/is-there-a-ruby-or-ruby-ism-for-not-nil-opposite-of-nil-method

Reply:
done